R/calc_F.R

```diff
@@ -74,10 +74,11 @@ get_F <- function(timeseries, fleetnames) {
     # feed back as a named vector sorted by fleet, then season. Names are the
     # same as in the PARAMETERS section of report.sso
     init_F <- init_F[order(init_F[, "Fleet"], init_F[, "Seas"]), ]
-    fleetnames_df <- data.frame(
-      Fleet = seq_along(fleetnames),
-      fleetname = fleetnames
-    )
+    
+    # edit to account for fleets of type==3 in list of fleets
```

Comment:
I want to make sure I follow this. It looks like Fleet type = 3 is a survey (thanks to the [ss3 docs](https://nmfs-ost.github.io/ss3-doc/SS330_User_Manual_release.html#fleet-definitions). If any survey fleets (which have no F) appear before fleets of other types, the numbering will be thrown off?

Reply:
Yes -- RS had fleets of type 3 mixed in with fleets of type 1 and 2. "fleetnames" input was subsetted to only include type=1 and 2, then was renumbering the subsetted fleetnames and applying the label that corresponded to this new number to create fleetnames_df. For RS, when fleets of type 3 were removed, the new renumbering in fleetnames_df no longer matched the true fleet number of the non-subsetted fleetnames_all, so the wrong label was being applied to any fleet that came after a fleet of type3.